doc/locale/ja/LC_MESSAGES/contribution/development/build/windows_cmake.po

```diff
@@ -127,9 +127,9 @@ msgid "``-DCMAKE_INSTALL_PREFIX=C:\\Groonga``"
 msgstr ""
 
 msgid ""
-"It specifies that you install your Groonga into ``C:\\\\Groonga`` folder."
+"It specifies that you install your Groonga into ``C:\\Groonga`` folder."
```

Comment:
I don't think this English explain what you intend. 
If I translate it will be
「あなたのGroongaを‘‘C:\\Groonga‘‘folderにインストールすことは明示する」
and don' think will make seance.
Because "It V that-" structure take "that-" as the subject.

So you probably want to use the specific command instead of using "It" as Subject.

Probably....
This command specifies that ~
(IF you had talked about the command before this sentence, you may use "it" but is is still bit confusing. I think)

Reply:
How about the following sentence?

`It specifies ``C:\Groonga`` as a folder to install your Groonga.`

I'll change it like the above for now, but feel free to give your opinion.

Reply:
I think your new sentence does more make sense.